tests/testthat/test_distToCells.R

```diff
@@ -0,0 +1,595 @@
+test_that("distToCells works",{
```

Comment:
I think here you just copied the tests four times depending on the metric, which is okay but not totally necessary. Would be good to have some tests that numerically test the differences between all the metrics for a given image.